src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -35,10 +39,10 @@ secondary = true
 
 # Main dependencies
 [tool.poetry.dependencies]
-python = ">=3.7, <3.9"
+python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
-ansys-saf-glow = {version = "0.1.9", source = "solutions-private-pypi" }
-ansys-saf-portal = {version = "0.1.4", source = "solutions-private-pypi"}
+ansys-saf-glow = {version = "^0.2", source = "solutions-private-pypi" }
+ansys-saf-portal = {version = "^0.1.6", source = "solutions-private-pypi"}
 toml = {version = "*"}
 tox = {version = "*"} 
```

Comment:
@ansJBoly please constraint the versions of ``toml`` and ``tox``.

Reply:
@iazehaf done